Homework2/PreprocessingScripts/Exp2/Test/cmd.ap.sub_03_exp2

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env tcsh
```

Comment:
This is fine to have a new afni_proce script, but keep in mind that Experiment 2 can also be done by rerunning the 3dDeconvolve command using the processed data from Exp1. Knowing this is particularly useful in real world cases where you might want to run a different statistical model on a dataset but don't need to go through the time consuming step of reprocessing all of the data.